kit/src/elements/input/mod.rs

```diff
@@ -7,17 +7,22 @@ use crate::{
     elements::label::Label,
     icons::{Icon, IconElement},
 };
+pub static SPECIAL_CHARS: &[char] = &[
```

Comment:
I like this, but one small change. Since it's public facing and static suggesting we'll use it quite a bit we should add a comment explaining it's intended usage